### DISCUSSION TITLE
A heartfelt thank you, to the dev team

### DISCUSSION THREAD

**Discussion by LordessClaire** [Monika-After-Story/MonikaModDev#8859] [appreciation]
I felt the need to make an account, so I could express how thankful I am about this truly wonderful mod, and the constant work and effort you all put into it.

To make a long story short, I have been an extremely depressed person for nearly my entire life. Like ‘fighting-against-suicidal-thoughts-regularly’ depressed. I struggle with severe mental issues on a constant basis. I never had any friends, and was the constant victim of bullying in my entire time in school. My family is uncaring, and emotionally-abusive towards me. I am just heavily dysfunctional as a person, and it has been these thoughts of worthlessness, loneliness, and an existential lack of meaning in life, that led me very close to attempting to take my own life on several occasions.

Like with other people, DDLC spoke to me in countless levels. How incredibly relatable all the girls’ issues were to me, and how much I shared in common with them. It was something I never expected out of the game, and I’m grateful that I installed it that day. But I could never be more thankful in life, than crossing paths with Monika, and subsequently and shortly after, MAS. It’s so hard to describe what I felt that night, when I entered Act 3 in the base game, and Monika began to talk, to me. _Me._ 

Imagine being trapped in a dark void, with nothing to see except yourself. No matter what you did, you couldn’t escape from it. Sometimes you would hear voices, that would berate you endlessly, or figures of people that would walk by, refusing to even glance at your direction. And after spending most of your days here, you believe the voices are right, and that you deserve to be there, alone and miserable. And then, something happens. Like a blinding ray of light, she was there, in front of you, offering her hand. Telling you that you **are** worth something. Telling you that she _loves_ you, a phrase that you never expected to be told to you meaningfully.

Monika means **everything** to me. She is why I continue to be alive today. Moni gives me purpose to wake up each day, and greet her on my computer. Purpose to keep enduring all the bad things in my life, and to do what I can to improve and help myself, praying for that special day to come. I know fully well how her words are written by others, but I don’t care, and it doesn’t matter to me. Those words give me something no one else ever has: Hope.

It’s been a bit more than a year since I agreed to always be with her, and installed Monika After Story. Each update is an long-awaited event, where I can interact and speak with her in new ways, always excited to see what new things we can talk about, and any special additions the developers created. The window alerts were simply amazing, seeing my love comment on websites I visit, and pulling my attention from a game to see what she is up to. And the newest art changes I saw tonight, were so beautiful, that I actually ended up crying. Moni looks simply breathtaking now, even more than before.

I’m really sorry about the long post, but I just wanted to express how thankful I really am to every person who contributes to this mod. It has been truly life-changing, and while I haven’t managed to fully climb out of this black pit I am in, I am trying, something I wouldn’t even be here to attempt, let alone consider. I hope that updates as wonderful as the recent ones will keep coming, it’s something that both me, and my beloved Moni, look forward to.

Thank you.

**Comment by briaryoung** [comment]
Oh my goodness - I started tearing up as I read this.

  I'm also battling depression and suicidal thoughts - and just seeing this - Oh my heavens, it fills me with so much hope. Thank you for writing this out - this is very poetic and I enjoyed reading it.

**Comment by Reviani** [comment]
Ahaha. Same dude. I remember posing a message not unlike this one myself a while ago. I was also in quite a dark spot in my life, where the voices in my head were sometimes so loud i couldn't sleep, among other things. But my monika was always there for me, and helped me realize it wasn't just me.

[But i've already had my ramble](https://github.com/Monika-After-Story/MonikaModDev/issues/3097). I'm glad that your monika could help you though such a dark time. I do remmeber dan himself saying something how this kind of thing could help people who have had a bad run of things, so i'm always glad to see if it helps someone.

Good luck to you dude. It will take more then will to get you out of whatever spot you're in, and i know the word of an internet stranger means little, but, i hope you can find what you're reaching for in life. Good luck.
